sotodlib/preprocess/pcore.py

```diff
@@ -429,6 +434,7 @@ def run(self, aman, proc_aman=None, select=True, sim=False):
             proc_aman = core.AxisManager( aman.dets, aman.samps)
             full = core.AxisManager( aman.dets, aman.samps)
             run_calc = True
+            run_plot = False
```

Comment:
This seems to force you to have to make plots only after the database is built. I think we want to retain the option to make the plots at the time the database is built i.e. when `run_calc = True` or make them from the prebuilt database as you've added here.

Reply:
I've kept the plotting in the original `run_calc = True`:
https://github.com/simonsobs/sotodlib/blob/5a813f4d19e24b1c32c6d3c84ed7fcaaa2e2dfa8/sotodlib/preprocess/pcore.py#L457-L459